### PR TITLE
Style fix

### PR DESCRIPTION
**Pull request by crgodfirnon**
Changed tabs to spaces. Put brackets around if/else if either of the if/else are blocks.

### CODE REVIEW

File: src/FileSystemReceiver/WindowsFileSystemReceiver.cpp [hunk 1]
```diff
@@ -3,48 +3,50 @@
...
         {
             // 1) grab substrings until next \ (keep trailing '\')
             int endPos = dir.find('\\');
-            std::string nextLocation = endPos == std::string::npos ? dir : dir.substr(0, endPos + 1);
-            if(nextLocation.back() != '\\') nextLocation.push_back('\\');
-            //std::cout << "nextLocation: " << nextLocation << std::endl;
-            dir = endPos == std::string::npos ? "" : dir.substr(endPos + 1);
+            std::string nextLocation = (endPos == std::string::npos) ? dir : dir.substr(0, endPos + 1);
+            
+            if (nextLocation.back() != '\\')
```
Comment (rjesquivias) [inline src/FileSystemReceiver/WindowsFileSystemReceiver.cpp]: {

}

File: src/FileSystemReceiver/WindowsFileSystemReceiver.cpp [hunk 2]
```diff
@@ -53,24 +55,22 @@ ReturnTypes WindowsFileSystemReceiver::changeDirectory(std::string dir)
                     }
                 }
 
-				//std::cout << "position where we found \\: " << endPos << std::endl;
                 // Safely ignore if we are at the root drive
-                if(endPos == 0) continue;
+                if (endPos == 0)
```
Comment (rjesquivias) [inline src/FileSystemReceiver/WindowsFileSystemReceiver.cpp]: {

}

File: src/FileSystemReceiver/WindowsFileSystemReceiver.cpp [hunk 3]
```diff
@@ -82,15 +82,16 @@ ReturnTypes WindowsFileSystemReceiver::changeDirectory(std::string dir)
 
 ReturnTypes WindowsFileSystemReceiver::execute(std::string file)
 {
-    if(file.empty()) return ReturnTypes::INVALID_INPUT;
+    if (file.empty())
```
Comment (rjesquivias) [inline src/FileSystemReceiver/WindowsFileSystemReceiver.cpp]: {

}

File: src/FileSystemReceiver/WindowsFileSystemReceiver.cpp [hunk 4]
```diff
@@ -82,15 +82,16 @@ ReturnTypes WindowsFileSystemReceiver::changeDirectory(std::string dir)
...
 
-    bool status = fileSystem->isAbsolutePath(file) ? fileSystem->execute(file) : fileSystem->execute(getDirectory() + file);
+    bool status = this->fileSystem->isAbsolutePath(file) ? this->fileSystem->execute(file) : this->fileSystem->execute(getDirectory() + file);
     return status ? ReturnTypes::SUCCESS : ReturnTypes::FAILURE;
 }
 
 ReturnTypes WindowsFileSystemReceiver::getFile(std::string fileName) const
 {
-    if(!fileSystem->isAbsolutePath(fileName))
+    if (!this->fileSystem->isAbsolutePath(fileName))
```
Comment (rjesquivias) [inline src/FileSystemReceiver/WindowsFileSystemReceiver.cpp]: {

}

File: test/src/Communicator/MockCommunicator.cpp [hunk 5]
```diff
@@ -4,7 +4,9 @@ ReturnTypes MockCommunicator::send(char* data, int size)
 {
     for(int i = 0; i < size; i++)
     {
-        if(historyIndex >= getBufferSize()) historyIndex = 0; // Loop back around and start rerecording data
+        // Loop back around and start rerecording data
+        if(historyIndex >= getBufferSize())
```
Comment (rjesquivias) [inline test/src/Communicator/MockCommunicator.cpp]: {

}

File: test/src/FileSystemReceiver/FileSystem/MockWindowsFileSystem.cpp [hunk 6]
```diff
@@ -2,11 +2,11 @@
 
 bool MockWindowsFileSystem::isDirectory(std::string directory) 
 { 
-    if(directory.empty()) return false;
+    if(directory.empty())
```
Comment (rjesquivias) [inline test/src/FileSystemReceiver/FileSystem/MockWindowsFileSystem.cpp]: {

}

File: test/src/FileSystemReceiver/FileSystem/MockWindowsFileSystem.cpp [hunk 7]
```diff
@@ -28,31 +28,36 @@ the path is relative to the current directory.
 */
 bool MockWindowsFileSystem::isAbsolutePath(std::string dir)
 {
-    if(dir.empty()) return false;
+    if(dir.empty())
```
Comment (rjesquivias) [inline test/src/FileSystemReceiver/FileSystem/MockWindowsFileSystem.cpp]: {

}

File: test/src/FileSystemReceiver/FileSystem/MockWindowsFileSystem.cpp [hunk 8]
```diff
@@ -28,31 +28,36 @@ the path is relative to the current directory.
...
     if(isalpha(dir[0]))
     {
         if(dir.size() >= 2)
         {
-            if(dir[1] != ':') return false; // Must start with "C:" or "D:" etc...
-
-            if(dir.size() >= 3) // Strange edge case of form: C:Projects\apilibrary\apilibrary.sln (Relative)
-                if(dir[2] != '\\') return false;
+            // Must start with "C:" or "D:" etc...
+            if(dir[1] != ':')
```
Comment (rjesquivias) [inline test/src/FileSystemReceiver/FileSystem/MockWindowsFileSystem.cpp]: {

}

File: test/src/FileSystemReceiver/FileSystem/MockWindowsFileSystem.cpp [hunk 9]
```diff
@@ -28,31 +28,36 @@ the path is relative to the current directory.
...
-            if(dir[1] != ':') return false; // Must start with "C:" or "D:" etc...
-
-            if(dir.size() >= 3) // Strange edge case of form: C:Projects\apilibrary\apilibrary.sln (Relative)
-                if(dir[2] != '\\') return false;
+            // Must start with "C:" or "D:" etc...
+            if(dir[1] != ':') 
+                return false;
+
+            // Strange edge case of form: C:Projects\apilibrary\apilibrary.sln (Relative)
+            if(dir.size() >= 3)
```
Comment (rjesquivias) [inline test/src/FileSystemReceiver/FileSystem/MockWindowsFileSystem.cpp]: {

}

File: test/src/FileSystemReceiver/FileSystem/MockWindowsFileSystem.cpp [hunk 10]
```diff
@@ -28,31 +28,36 @@ the path is relative to the current directory.
...
+                if(dir[2] != '\\')
```
Comment (rjesquivias) [inline test/src/FileSystemReceiver/FileSystem/MockWindowsFileSystem.cpp]: {

}

File: test/src/FileSystemReceiver/FileSystem/MockWindowsFileSystem.cpp [hunk 11]
```diff
@@ -81,22 +86,28 @@ int MockWindowsFileSystem::readIntoBuffer(char* buffer, int bufSize, int readPos
             fileFound = true;
     }
 
-    if(!fileFound) return -1; // TODO: throw file not found error
+    // TODO: throw file not found error
+    if(!fileFound)
```
Comment (rjesquivias) [inline test/src/FileSystemReceiver/FileSystem/MockWindowsFileSystem.cpp]: {

}

File: test/src/FileSystemReceiver/FileSystem/WindowsFileSystemTest.cpp [hunk 12]
```diff
@@ -55,12 +55,12 @@ TEST(WindowsFileSystemTest, TestIsAbsolutePath)
...
-    std::string relative3 = "C:Projects\\apilibrary\\apilibrary.sln";	            // A relative path from the current directory of the C: drive.
+    std::string absolute1 = "C:\\Documents\\Newsletters\\Summer2018.pdf";               // An absolute file path from the root of drive C:
+    std::string absolute2 = "\\Program Files\\Custom Utilities\\StringFinder.exe";      // An absolute path from the root of the current drive.
+    std::string relative1 = "2018\\January.xlsx";                                       // A relative path to a file in a subdirectory of the current directory.
+    std::string relative2 = "..\\Publications\\TravelBrochure.pdf";                     // A relative path to file in a directory that is a peer of the current directory.
+    std::string absolute3 = "C:\\Projects\\apilibrary\\apilibrary.sln";                 // An absolute path to a file from the root of drive C:
+    std::string relative3 = "C:Projects\\apilibrary\\apilibrary.sln";                   // A relative path from the current directory of the C: drive.
```
Comment (rjesquivias) [inline test/src/FileSystemReceiver/FileSystem/WindowsFileSystemTest.cpp]: How are you finding these?

File: test/src/FileSystemReceiver/WindowsFileSystemReceiverTest.cpp [hunk 13]
```diff
@@ -8,157 +8,158 @@
 TEST(WindowsFileSystemReceiverTest, TestChangeDirectoryCommand)
 {
     std::shared_ptr<IFileSystem> fileSystem = std::make_shared<MockWindowsFileSystem>();
-	std::shared_ptr<ICommunicator> communicator = std::make_shared<MockCommunicator>();
```
Comment (rjesquivias) [inline test/src/FileSystemReceiver/WindowsFileSystemReceiverTest.cpp]: These are really weird.  For some reason the formatting is good on my end so I'm not sure why its coming out different here.

File: src/FileSystemReceiver/FileSystem/WindowsFileSystem.cpp [hunk 14]
```diff
@@ -2,24 +2,32 @@
 
 bool FileSystem::execute(std::string file)
 {
-    if(!system(NULL)) return false;
+    if (!system(NULL))
```
Comment (rjesquivias) [inline src/FileSystemReceiver/FileSystem/WindowsFileSystem.cpp]: Add this style everywhere else I commented in the file where needed.

File: src/FileSystemReceiver/WindowsFileSystemReceiver.cpp [hunk 15]
```diff
@@ -100,7 +101,7 @@ ReturnTypes WindowsFileSystemReceiver::getFile(std::string fileName) const
     do
     {
         // use the filesystem interface to read data into our buffer
-        bytesRead = fileSystem->readIntoBuffer(buffer, bufSize, readPosition, fileName);
+        bytesRead = this->fileSystem->readIntoBuffer(buffer, bufSize, readPosition, fileName);
         if(bytesRead == -1) return ReturnTypes::INVALID_FILENAME;
         readPosition += bytesRead;
```
Comment (rjesquivias) [inline src/FileSystemReceiver/WindowsFileSystemReceiver.cpp]: this->communication->send(buffer, bytesRead);

Comment (rjesquivias) [reply src/FileSystemReceiver/WindowsFileSystemReceiver.cpp]: ahh OF COURSE it shows the code above where i wanted this comment to be lol.

File: test/src/FileSystemReceiver/FileSystem/WindowsFileSystemTest.cpp (hunk 12 above)
Comment (crgodfirnon) [reply test/src/FileSystemReceiver/FileSystem/WindowsFileSystemTest.cpp]: Visual studio tools -> text editor -> General and enable view Tab/Space

Comment (crgodfirnon) [reply test/src/FileSystemReceiver/FileSystem/WindowsFileSystemTest.cpp]: Then you want to like always replace tabs with 4 spaces
